style.css

```diff
@@ -168,6 +168,106 @@
     list-style-type: none;
   }
 
+  .pop-up-modal {
+    position: absolute;
+    background-color: white;
+    margin-top: 70px;
+    border-radius: 0.5rem;
+    width: 375px;
+    height: auto;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+
+  .overlay {
+    display: none;
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    overflow: auto;
+    background-color: rgba(0, 0, 0, 0.8);
+  }
+
+  .img-cont-pop {
+    display: block;
+    text-align: center;
+    padding-top: 10px;
+  }
+
+  #img {
+    width: 350px;
+    height: auto;
+  }
+
+  .close-btn-cont {
+    display: flex;
+    justify-content: flex-end;
+    height: 20px;
+    width: 350px;
+    margin: 200px 20px 0 0;
```

Comment:
- [ ] You did an amazing job following the requested changes from the previous reviewer but please stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468). Kindly put the x button more to the right following the design. Keep rocking 💪 

Here you have a screenshot of your button:

![image](https://user-images.githubusercontent.com/74506933/156473338-eec7cc91-dcbe-48b7-a390-03811c758823.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/156473353-4ad16ae1-06e3-42f2-82d2-ac27afdbb499.png)

Reply:
thank you, now it's just align with the image